python/nnvm/top/nn.py

```diff
@@ -192,3 +192,49 @@ def schedule_upsampling(_, outs, target):
         return topi.generic.schedule_injective(outs)
 
 reg.register_pattern("upsampling", OpPattern.INJECTIVE)
+
+@reg.register_compute("shortcut")
+def compute_shortcut(attrs, inputs, _):
+    """Compute definition of shortcut"""
+    out = topi.nn.vision.shortcut(inputs[0], inputs[1])
+    return out
+
+@reg.register_schedule("shortcut")
+def schedule_shortcut(attrs, outs, target):
+    """Schedule definition of shortcut"""
+    with tvm.target.create(target):
+        return topi.generic.schedule_shortcut(outs)
+
+reg.register_pattern("shortcut", OpPattern.OUT_ELEMWISE_FUSABLE)
+
+@reg.register_compute("reorg")
+def compute_reorg(attrs, inputs, _):
+    """Compute definition of reorg"""
+    return topi.nn.vision.reorg(inputs[0], attrs.get_int("stride"))
+
+@reg.register_schedule("reorg")
+def schedule_reorg(attrs, outs, target):
+    """Schedule definition of reorg"""
+    with tvm.target.create(target):
+        return topi.generic.schedule_reorg(outs)
+
+reg.register_pattern("reorg", OpPattern.OUT_ELEMWISE_FUSABLE)
```

Comment:
Make sure you set the type correctly. If we set type to be OUT_ELEMWISE_FUSABLE, it likely means the schedule should deal with case when further outputs get fused back. If the schedule cannot deal with such case, set it to be OPAQUE.